shared/actions/chat.js

```diff
@@ -86,6 +86,77 @@ const {
 
 const {conversationIDToKey, keyToConversationID, keyToOutboxID, InboxStateRecord, MetaDataRecord, makeSnippet, outboxIDToKey, serverMessageToMessageBody, getBrokenUsers} = Constants
 
+// Whitelisted action loggers
+const loadedInboxActionTransformer = action => ({
+  payload: {
+    inbox: action.payload.inbox.map(i => {
+      const {
+        conversationIDKey,
+        muted,
+        time,
+        unreadCount,
+        validated,
+        participants,
+        info,
+      } = i
+
+      return {
+        conversationIDKey,
+        info: {
+          status: info && info.status,
+        },
+        muted,
+        participantsCount: participants.count(),
+        time,
+        unreadCount,
+        validated,
+      }
+    }),
+  },
+  type: action.type,
+})
+
+const postMessageActionTransformer = action => ({
+  payload: {
+    conversationIDKey: action.payload.conversationIDKey,
+  },
+  type: action.type,
+})
+
+const retryMessageActionTransformer = action => ({
+  payload: {
+    conversationIDKey: action.payload.conversationIDKey,
+    outboxIDKey: action.payload.outboxIDKey,
+  },
+  type: action.type,
+})
+
+const safeServerMessageMap = m => ({
+  key: m.key,
+  messageID: m.messageID,
+  messageState: m.messageState,
+  outboxID: m.outboxID,
+  type: m.type,
+})
+
+const prependMessagesActionTransformer = action => ({
+  payload: {
+    conversationIDKey: action.payload.conversationIDKey,
+    messages: action.payload.messages.map(safeServerMessageMap),
+    moreToLoad: action.payload.moreToLoad,
+    paginationNext: !!action.payload.paginationNext,
```

Comment:
:sheep: Perhaps name this something like "paginationNextIsTruthy" to indicate that it's not the true value of `paginationNext`? If I were inspecting logs this could confuse me for a moment.